Earthfile

```diff
@@ -284,12 +283,15 @@ earthly-docker:
 earthly-integration-test-base:
     FROM +earthly-docker
     ENV EARTHLY_CONVERSION_PARALLELISM=5
-    RUN earthly config global.disable_analytics true
-    RUN earthly config global.local_registry_host 'tcp://127.0.0.1:8371'
+    ENV GLOBAL_CONFIG={disable_analytics: true, local_registry_host: 'tcp://127.0.0.1:8371'}
+    ENV NO_DOCKER=1
+    ENV SRC_DIR=/test
+    ENV NETWORK_MODE=host
```

Comment:
Moved from the base image config since its our tests that need `host`, not the base image (I've been able to run `+for-linux` under the image without using host networking).